src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --main-color: #00acdc;
+  --background-color: #fff;
+  --text-color: #000;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  font-style: normal;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: var(--background-color);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  position: relative;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: block;
+  position: relative;
+  padding: 23px 0;
```

Comment:
it's bad practice to calculate every pixel: 
   for example if someday you decided to have your line-height bigger you should calculate paddings again 
I suggest it's better to set height to 60px and make the element flex to center  the content inside (your text)